test/verify/check-machines

```diff
@@ -1390,6 +1390,16 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
                                                       os_name=config.FEDORA_28,
                                                       start_vm=False))
 
+        virtInstallVersion = self.machine.execute("virt-install --version")
+        if virtInstallVersion >= "2":
```

Comment:
I'm not a big fan of such dynamic checks. They get silently skipped if the condition is somehow wrong, or the output format changes, etc. It'd be better to have an explicit list of OSes where it does *not* work, and also cover the "else:" behaviour.

Reply:
Fixed. Also fixed in another place which followed this approach.